### ISSUE TITLE
Buglist

### ISSUE THREAD

**Issue by Grantelbart4** [OpenRA/ra2#227]
I thought it would be a good idea to mention all bugs in one single list:

- [ ] war miners don't shoot at enemies when they are mining
- [x] british snipers only attack units which attack them #263
- [ ] kirovs attack buildings from a distance where they do no damage at all
- [ ] kirovs don't automatically attack
- [ ] only one plane can be put on an airfield #87
- [ ] planes stop when flying over cliffs
- [ ] ships and submarines can be seen through cliffs https://github.com/OpenRA/OpenRA/issues/7520
- [ ] Prism towers don't attack together #133
- [ ] tank shells fly way too fast. They can't be seen by the naked eye
- [ ] testlatroopers fire they bolt from their feet instead of their gun
- [ ] paratroopers drop far away from the point where they land
- [x] burning and electrified enemies die with the wrong palette #228
- [ ] crouching enemies seem to make pushups
- [ ] paradropping planes don't fly back after dropping paratroopers
- [ ] units don't attack enemies with a higher range than them when being attacked
- [ ] destroyers can NOT attack submarines when submerged
- [ ] mirage tanks have no stealth function although it is technically possible (e.g. ra1 sniper)
- [ ] Night Hawk stops at the momentary altitude when ordered to attack. Can't attack at certain altitudes
- [ ] enemy buildings can't be captured by and engineer when they get repaired
- [ ] kirovs drop normal bombs instead of kirov bombs
- [ ] Cargo Plane's health is invisible. I think this is a problem that should be solved on upstream, but for vanilla RA2's unselectable units you can see its health by hovering cursor on it. (See #198.)

**Comment by GraionDilach** [comment]
Mirage Tank had a special disguise and not stealth.

**Comment by scshunt** [comment]
It is important to file bugs separately, so that we can track when each one is fixed and discuss each one separately. A single issue makes it hard to follow all of the individual issues. Please file an issue for each separate bug.

**Comment by Elthan** [comment]
Could be an idea to have a list linking to all the separate bugs? I agree that each bug should be its own issue though.

**Comment by Grantelbart4** [comment]
I am currently working on fixing the building offets, but I can't work with all the others as I don't understand the game logic. But seriously? You want me to create a thousand threads? Sure, go ahead.

**Comment by Mailaender** [comment]
Went through the list. While it is informative it contains a lot of duplicates or engine level problems. About half of it is fixed by now.
